build.sbt

```diff
@@ -577,10 +568,32 @@ lazy val `scio-avro`: Project = project
     )
   )
   .dependsOn(
-    `scio-core` % "compile;it->it"
+    `scio-core` % "compile;it->it",
+    `scio-avro-base`
   )
   .configs(IntegrationTest)
 
+lazy val `scio-avro-base`: Project = project
```

Comment:
Looks good to me. The only suggestion here would be to keep this named as `scio-avro`. I reckon that users not using the typed API will not need to update the dep.

Reply:
If we do that it becomes a breaking change since people will need to update their dependencies. If we're okay with that I can make to suggested change. @nevillelyh wdyt ?